app/views/admin/posts/new_user.js.erb

```diff
@@ -0,0 +1,2 @@
+$('#change-user-modal-<%= @post_id %> .search-results').html("<%= j render 'admin/posts/new_user_form', post_id: @post_id %>");
+$('#new_user').enableClientSideValidations();
```

Comment:
This needs to call the global `Helpy.ready();` function after it appends the form, or else the disabled listener does not get added to the new form.